locales/uk.json

```diff
@@ -1677,12 +1677,12 @@
 			"atSchool": "у школі",
 			"atYourSchool": "у вашій школі ",
 			"created": "створено",
-			"from": "з",
+			"by": "",
```

Comment:
is it ok that there is no translation in ukrainan for this?

Reply:
yes it is ok this is the translation I got from Viktoria, she is Ukrainian